### ISSUE TITLE
hello, a question, can I use my own dataset labeled by labelme to train the model??

### ISSUE THREAD

**Issue by zhengkai17** [jfzhang95/pytorch-deeplab-xception#100]
hello, a question, can I use my own dataset labeled by labelme to train the model??  thx

**Comment by iariav** [comment]
yes, you need to change mypath.py and add a class for loading your dataset in 'dataloaders/datasets'.
you might also need to add some dataset related functions to 'dataloaders/utils.py'

**Comment by LoneWolf1999** [comment]
hello, can you specify related functions to train our own models ?

**Comment by kungkook** [comment]
> hello, a question, can I use my own dataset labeled by labelme to train the model?? thx

I have the same problem. Have you solved it

**Comment by karryor** [comment]
hello,how do you build your dataset file folder？
